setup.py

```diff
@@ -32,6 +32,12 @@
                     if not line.startswith('#')]
 
 
+extras_require = {
+    'test fixtures': ['caproto', 'curio', 'pytest >=3.9', 'trio'
```

Comment:
I think this should have spaces in it. Can we do `test_fixtures`?

Reply:
Also, this list is missing `attrs >=18.1.0`.

Reply:
done